sim/target-agnostic.mk

```diff
@@ -39,18 +39,15 @@ PLATFORM ?=
 DRIVER_CC ?=
 DRIVER_H ?=
 
-# Defined for each platform
-platforms_dir := $(abspath $(firesim_base_dir)/../platforms)
-vitis_CXX_FLAGS ?= -std=c++14 -idirafter ${CONDA_PREFIX}/include -idirafter /usr/include -idirafter $(XILINX_XRT)/include
-vitis_LD_FLAGS ?= -L${CONDA_PREFIX}/lib -Wl,-rpath-link=/usr/lib/x86_64-linux-gnu -L$(XILINX_XRT)/lib -luuid -lxrt_coreutil
-f1_CXX_FLAGS ?= -std=c++11 -I$(platforms_dir)/f1/aws-fpga/sdk/userspace/include
-f1_LD_FLAGS ?=
-
 # Target-specific CXX and LD flags for compiling the driver and meta-simulators
+# These should be platform independent should be governed by the target-specific makefrag
```

Comment:
I'm confused by this comment because it looks like you're moving what we're doing here into the targets defined in `target-agnostic.mk`, not into the target-specific makefrags.

Reply:
I'm missing an "and" here. These were intended to give a mechanism for target-specific makefrags to glom on settings that are then passed to all driver/metasimulator targets.